lib/livebook_web/live/output/file_input_component.ex

```diff
@@ -54,7 +54,7 @@ defmodule LivebookWeb.Output.FileInputComponent do
             Click to select a file or drag a local file here
           <% end %>
         </div>
-        <.live_file_input upload={@uploads.file} class="hidden" accept={@accept} />
+        <.live_file_input upload={%{@uploads.file | accept: @accept}} class="hidden" />
```

Comment:
@chrismccord since LV 0.19 passing `accept` is unexpected. In our specific case we can't pass accept to `allow_upload`, because it is user-specified (in `Kino.Input.file/2`), so it may include unknown MIME and `allow_upload` would raise. I think the override here is fine, but pointing out just in case.

Reply:
Whoops, it's a tuple, but I will adjust.

Reply:
The warning looks like false positive actually

```
warning: attribute "upload" in component Phoenix.Component.live_file_input/1 must be a {:struct, Phoenix.LiveView.UploadConfig}, got: %{@uploads.file | accept: @accept}
```

Using a function prevents it.